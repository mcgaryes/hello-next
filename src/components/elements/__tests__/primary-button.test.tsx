import {render} from '@testing-library/react'
import PrimaryButton from "@/elements/primary-button";

test('Renders primary-button unchanged', () => {
    const {container} = render(<PrimaryButton />)
    expect(container).toMatchSnapshot()
})
