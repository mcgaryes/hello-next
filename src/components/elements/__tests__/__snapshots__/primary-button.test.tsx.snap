// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders primary-button unchanged 1`] = `
<div>
  <div>
    <button
      class="bg-blue-500 px-4 py-2 rounded-md text-white hover:bg-blue-600 active:bg-blue-800"
    />
  </div>
</div>
`;
